### ISSUE TITLE
Communities: The pointer is shown and doesn't work for an empty category

### ISSUE THREAD

**Issue by elina2015** [status-im/status-desktop#9300]
# Bug Report

## Steps to reproduce
1. Create a community and add there an empty category

#### Expected behavior
The pointer is not shown
#### Actual behavior
The pointer is shown and doesn't work for an empty category
<img width="152" alt="image" src="https://user-images.githubusercontent.com/14942081/214757482-09313eca-d3d1-4c77-a963-63ed4134ab74.png">

### Additional Information

- Status desktop version: master
- Operating System: macOS Monterey

**Comment by 0x-r4bbit** [comment]
I will changes that according to the issue description but I want to clarify that this is actually the correct behaviour.
It's an empty category, so there's not items in it but it's still a collapsible item.

**Comment by jrainville** [comment]
> I will changes that according to the issue description but I want to clarify that this is actually the correct behaviour. It's an empty category, so there's not items in it but it's still a collapsible item.

Yeah it makes sense to remove the arrow since it doesn't have items.

Let me know if you need help with how the new chat model works. There are no longer category items on the Nim side. just chat items with category data attached

**Comment by 0x-r4bbit** [comment]
Stopping the work on this.
I put in quite some time to figure out the current chats/categories situation and I find it super hard to follow what's going on and how those categories are managed in the model.

@jrainville not offense. I pinged you directly about this. It's probably more efficient if you add this flag as you're the only one who's familiar with the new flattened implementation

**Comment by 0x-r4bbit** [comment]
Gave this another shot after reading @jrainville new docs on how the chat models is implemented (https://hackmd.io/@status-desktop/Sk95Jct2i).
Turns out I was 98% there.
Thanks for putting this together.

**Comment by 0x-r4bbit** [comment]
Closing this as of https://github.com/status-im/status-desktop/pull/9420#issuecomment-1480743410
